spec/PluginClientSpec.php

```diff
@@ -87,47 +87,4 @@ function it_throws_loop_exception(HttpClient $httpClient, RequestInterface $requ
 
         $this->shouldThrow('Http\Client\Common\Exception\LoopException')->duringSendRequest($request);
     }
-
-    function it_injects_debug_plugins(HttpClient $httpClient, ResponseInterface $response, RequestInterface $request, Plugin $plugin0, Plugin $plugin1, Plugin $debugPlugin)
```

Comment:
I didn't know why I removed this test... We must keep it as the feature is still present and still usable.